packages/cli/src/commands/storybook.js

```diff
@@ -31,9 +31,21 @@ export const builder = (yargs) => {
       type: 'string',
       default: 'public/storybook',
     })
+    .option('no-manager-cache', {
+      describe:
+        'Do not cache the manager UI. Useful when you have made changes to `storybook.manager.js`.',
+      type: 'boolean',
+      default: false,
+    })
```

Comment:
Let's make this positive and then just flip it (easier to read). Also yargs automatically generates negatives for us, so doing it this way gives us a generated flag, `--no-manager-cache` with the value false.
```suggestion
    .option('manager-cache', {
      describe:
        'Cache the manager UI. Disable this when your making changes to `storybook.manager.js`.',
      type: 'boolean',
      default: true,
    })
```